templates/plugin/protocols.conf.erb

```diff
@@ -9,8 +9,8 @@ if @values
   end
 end
 -%>
-<% if @ignore_selected != nil -%>
-  IgnoreSelected <%= @ignore_selected %>
+<% if @ignoreselected != nil -%>
```

Comment:
Currently, `ignoreselected` is a Boolean.  That means it can never be `nil` (undef in puppet).

Reply:
Looks like it was always a boolean and never default to `undef`.